dbms/src/Storages/Page/V3/PageEntry.h

```diff
@@ -40,6 +40,20 @@ struct PageEntryV3
     PageFieldOffsetChecksums field_offsets{};
 
 public:
+    size_t getFieldSize(size_t index) const
+    {
+        if (unlikely(index >= field_offsets.size()))
+            throw Exception(fmt::format("Try to getFieldData of PageEntry [blob id={}] with invalid [index={}] [fields size={}]",
```

Comment:
```suggestion
            throw Exception(fmt::format("Try to getFieldData of PageEntry [blob_id={}] with invalid [index={}] [fields size={}]",
```
align the `blob_id` with other logging so that we can search log easier